### PR TITLE
Remove support for end-of-life Python and Django versions

### PR DESCRIPTION
**Pull request by jdufresne**
For a list of supported Python versions, see:
https://devguide.python.org/#status-of-python-branches

For a list of end-of-life Python versions, see:
https://devguide.python.org/devcycle/#end-of-life-branches

For a list of supported Django versions, see:
https://www.djangoproject.com/download/#supported-versions

By removing testing for end-of-life versions, we reduce testing time and
resources.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -33,7 +29,7 @@ classifiers =
 zip_safe = True
 py_modules = decorator_include
 install_requires = Django>=2.0
```
Comment (twidi) [inline setup.cfg:31]: 2.2

**Review by twidi — CHANGES_REQUESTED** [review]
Just one remark

**Comment by jdufresne** [comment]
Good catch. Fixed in the latest revision.

**Comment by twidi** [comment]
I guess we have to migrate from travis to another CI tool....  :-/

**Comment by twidi** [comment]
> I guess we have to migrate from travis to another CI tool.... :-/

I'll try to work on it this week-end, with circle-ci (or github actions but I never used it).

**Comment by jdufresne** [comment]
Agreed. While Travis is **super slow** it eventually finished. Thoughts on merging these PRs in the meantime?

**Review by twidi — APPROVED** [review]

**Pull request merged**